### ISSUE TITLE
Future of the installer (MSI or EXE)

### ISSUE THREAD

**Issue by riezebosch** [docker/for-win#954]
When digging through the *new* installer I stumbled upon the RSS containing latest [installer information](https://download.docker.com/win/edge/appcast.xml).

It didn't take long to find out this feed is also available for the [stable channel](https://download.docker.com/win/stable/appcast.xml). To my surprise there seems already a *new* .EXE installer available. Is this about to replace the .MSI installer altogether?

I'm asking because I currently maintain the [chocolatey package for docker-for-windows](chocolatey.org/packages/docker-for-windows). For the `-edge` installers I already switched to the new installer and I'm curious if I should do the same for stable.

In my experience the .EXE works fine on both `Windows 10` and `Windows Server 2016` although it is not mentioned on the [docker store download site](https://store.docker.com/editions/community/docker-ce-desktop-windows).

**Comment by jasonbivins** [comment]
Hi @riezebosch 
The .msi versions are legacy versions.  I'd advise using the .exe versions, unless you have install issues, in which case we'll work with you to sort them out.  We're maintaining them for the time being, but don't plan to keep them as a permanent installer option.

**Comment by riezebosch** [comment]
But why is there still the msi download on the docker store page then for the stable channel?

**Comment by friism** [comment]
@riezebosch the .exe installer is still experimental which is why we only document it for the edge channel

**Comment by riezebosch** [comment]
So for now it is better to still use the MSI for the stable channel package and the EXE for the edge channel I presume? Until you also switch over to the EXE for stable.

**Comment by jasonbivins** [comment]
Hi @riezebosch 
We actually just got an update regarding the installers this morning, and the .EXE installers are good for both versions.  We were originally only using this installer for the edge channel, but with the stable release of 17.06 we're cutting over to the .exe install.  The MSI will be available for the forseeable future though. 

 We just got updated on this today though, so the docs page hasn't been updated yet.  I hope this helps to clearn up any confusion!
